package.json

```diff
@@ -58,6 +58,7 @@
     "@dydxprotocol/v4-abacus": "1.10.2",
     "@dydxprotocol/v4-client-js": "^1.1.27",
     "@dydxprotocol/v4-localization": "^1.1.192",
+    "@dydxprotocol/v4-proto": "5.0.0-dev.0",
```

Comment:
v4-protos actually upgraded in v4-clients! we can bump this to match v4-clients:
 https://github.com/dydxprotocol/v4-clients/blob/main/v4-client-js/package.json#L39